### DISCUSSION TITLE
router.asPath different between SSR and SSG (on Vercel platform)

### DISCUSSION THREAD

**Discussion by hk86** [vercel/next.js#16256] [Help]
I use `router.asPath` in `_app` in order to set alternate link tags. i.e.: for the page `/pages/article/[id].js` SSR returns `/article/123`, but in case of SSG it return /article/[id]. I see this behavior on the Vercel platform, not locally.

I use Next.js 9.5.2.

**Comment by ijjk** [comment]
Hi, I opened a new issue to track this [here](https://github.com/vercel/next.js/issues/16269), thanks for reporting this it is being investigated!
